src/scripts/test_cli.py

```diff
@@ -170,12 +173,18 @@ def cli_config_tests(_tmp_dir):
     ldflags = test_cli("config", "ldflags")
     libs = test_cli("config", "libs")
 
-    if len(prefix) < 4 or prefix[0] != '/':
-        logging.error("Bad prefix %s", prefix)
+    if os.name == 'nt':
```

Comment:
For reference: Another (perhaps slightly more readable option):

```suggestion
    if platform.system() == 'Windows':
```

Reply:
changed